apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx

```diff
@@ -21,7 +26,24 @@ export const Home: FunctionComponent = () => {
           {firstName && ` ${firstName}`}
         </h3>
       </div>
-      <Outlet />
+      <div>
+        <Tabs defaultValue="statistics">
```

Comment:
The default value should be dependent on the URL. i.e. if the current route is `/:locale/workflows` then `defaultValue` should match that.